### PR TITLE
Export students

### PR DESCRIPTION
**Pull request by EvanZhouDev**
# Pull Request Check List

**Applicable issues**

#49 

**Description of PR**
 - [x] Exporting students
     - [x] Frontend/CSV handling
     - [x] Hook up backend
 - [x] Metadata modifications
 - [x] Your Last Attendance
     - [x] Dashboard quick-stats
     - [x] Modal w/ comprehensive stats
     - [x] Hook up backend

### CODE REVIEW

File: app/host/dashboard/components/YourLastAttendance.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                                </select>
+                            </label>
+                            <input
+                                type="text"
+                                placeholder="Search Attendees..."
+                                className="input input-standard ml-1 flex-grow"
+                                onChange={() => {}}
+                            />
+                        </div>
+                        <table className="table mt-5 w-full outline outline-base-200 outline-1 text-[#24292F] rounded-lg">
```
Comment (ThatXliner) [inline app/host/dashboard/components/YourLastAttendance.jsx:74]: Hard coded colors?

**Comment by ThatXliner** [comment]
@EvanZhouDev there is a glitch where the metadata UI won't update despite the `group` being updated via Realtime

**Review by ThatXliner — APPROVED** [review]
I recently changed some code to use `useCallback` (in the last commit). Please check my syntax. Merge when you're ready

**Comment by EvanZhouDev** [comment]
:shipit: LGTM

**Comment by EvanZhouDev** [comment]
Things to do:
- [ ] Actually get Attendance statistics working (like in Your Last Attendance)
- [ ] When adding a _attendee_, RLS complains about not being able to add a host to a group. However, by scanning code, it works
- [ ] Ending a session gives an error

**Comment by ThatXliner** [comment]
> For some reason, the realtime for the `attendee` table is broken. Specifically on insert

And I have no idea why.

**Comment by ThatXliner** [comment]
> > For some reason, the realtime for the `attendee` table is broken. Specifically on insert
> 
> And I have no idea why.

I found the issue, fixing

**Comment by ThatXliner** [comment]
So I'm just going to leave the attendee realtime scuffed for now (until we can find a better solution). Tomorrow I will need to

- Finish the GetLastAttendance, which when I'm done I will merge without your review
- Thoroughly test on role-seg
- Rewrite tests

**Pull request merged**